spec/requests/pingometer_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+RSpec.describe 'Pingometer Webhook', type: :request do
+  it 'is persisted appropriately in the database' do
+    monitor_id = 'b74014410cc1236a3d0h7400'
+
+    # Expect a pre-existing monitor
+    web_service = WebService.create pingometer_id: monitor_id,
```

Comment:
As noted before, a pingometer ID is really not a stable key. We’ve removed/rebuilt them several times in the past (resulting in new IDs). Hostname is much more stable. Alternatively, we’d need infrastructure to migrate monitors (probably automatically, since we’d never want to lose an event coming in real time between the time a change was made in Pingometer and when the application here was updated).